makefiles/docker.inc.mk

```diff
@@ -19,6 +32,25 @@ else
   export INSIDE_DOCKER := 0
 endif
 
+ifeq (0,$(INSIDE_DOCKER))
```

Comment:
Just noticed that this is executed even with `BUILD_IN_DOCKER=0` (just adding a second condition to the if would help).

Reply:
Good catch, fixed